test/unit/specs/store/Upload.spec.js

```diff
@@ -0,0 +1,56 @@
+import index from '../../../../src/store/modules/upload/index'
+
+const testAction = (action, args, state, expectedMutations, done) => {
+  let count = 0
+
+  // mock commit
+  const commit = (type, payload) => {
+    const mutation = expectedMutations[count]
+
+    try {
+      expect(mutation.type).to.equal(type)
```

Comment:
what exactly is this doing?

Reply:
the function will go through each mutation call by action and compare with the exceptedMutations

Reply:
Sweet! Thanks :)